src/agent/Makefile

```diff
@@ -53,6 +55,16 @@ ifeq ($(ARCH), aarch64)
     $(warning "WARNING: aarch64-musl needs extra symbols from libgcc")
 endif
 
+# The only architecture we can currently link the kata-agent
+# against glibc statically is x86_64.
+# Please, for more information refer to:
+# https://github.com/kata-containers/kata-containers/issues/1239
```

Comment:
@fidencio I have tested on s390x, this can be removed :)

Reply:
Oh yeah, I've removed the condition but not the comment. Removed now, thanks!